go.mod

```diff
@@ -467,6 +467,7 @@ replace (
 	modernc.org/strutil => modernc.org/strutil v1.0.0
 	modernc.org/xc => modernc.org/xc v1.0.0
 	sigs.k8s.io/kustomize => sigs.k8s.io/kustomize v2.0.3+incompatible
+	sigs.k8s.io/sig-storage-lib-external-provisioner => sigs.k8s.io/sig-storage-lib-external-provisioner v3.1.0+incompatible
```

Comment:
Is this not going to cause some circular dependency?

K8s requires translation requires external-lib requires k8s?

Reply:
`verify-no-vendor-cycles.sh` and `verify-vendor.sh` both pass just fine so I think this is ok...?

But you're right this does logically seem like a cycle. But also if what i've done here is ok i'd rather not jiggle the `VolumeOptions` object somewhere else and update all the pieces that use it. 

/cc @liggitt

Reply:
We have verify scripts that will fail on this cycle. k8s -> vendor -> k8s is not permitted

Reply:
If verify-no-vendor-cycles.sh doesn't fail on this PR then it is broken :(

Reply:
yup, it got broken in https://github.com/kubernetes/kubernetes/commit/e25e5a63e7b27e65b0118c3bfedfe1dec84c3f7c

Reply:
reverting/fixing in https://github.com/kubernetes/kubernetes/pull/77841